website/www/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "apache-beam-website",
+  "version": "1.0.0",
+  "description": "Apache Beam website",
+  "repository": "apache/beam",
+  "license": "MIT",
+  "scripts": {
+    "build_github_samples": "./build_github_samples.sh",
+    "develop": "cd site && hugo server",
+    "build": "cross-env HUGO_ENV=production hugo -d ../dist -s site -v",
+    "start": "hugo -d ../dist -s site -vw"
+  },
+  "dependencies": {},
+  "devDependencies": {
+    "autoprefixer": "^9.7.4",
+    "cross-env": "^7.0.2",
+    "postcss-cli": "^7.1.0"
```

Comment:
Is postcss/autoprefixer actually used anywhere? I can't find any other references to them

Reply:
I left them there during the initial Docsy setup because they're required, but it turns out that we don't use Webpack configuration. Removed!

Reply:
This is what I got on the build process when I tried to remove them. It's actually required by Docsy theme as a prerequisite. [1]
```
Error: Error building site: POSTCSS: failed to transform "scss/main.css" (text/css): resource "scss/scss/main.scss_60afefe949d523e83b9759c49b47fc3a" not found in file cache
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

[1] https://www.docsy.dev/docs/getting-started/#install-postcss

Reply:
Ah I see. Could you add a comment stating they're required for docsy?